src/components/results/dimension-based-results/dimensions/style.scss

```diff
@@ -1,10 +1,5 @@
 @import "~style/helpers";
 
 .dimensions {
-  font: 100%/1.5 "Source Sans Pro", "Helvetica Neue", Verdana, Arial, sans-serif;
-  font-weight: 400;
-  color: #333;
-  background: $white;
-  list-style: none;
```

Comment:
This `list-style: none;` needs to remain on this class, the dimension widgets are broken without it